src/Dev/Debug.php

```diff
@@ -195,53 +195,4 @@ protected static function supportsHTML(?HTTPRequest $request = null)
         }
         return false;
     }
-
-    /**
-     * Check if the user has permissions to run URL debug tools,
-     * else redirect them to log in.
-     */
-    public static function require_developer_login()
```

Comment:
This is removing public API best beta so would need approval, and a corresponding CMS 5 deprecation

Reply:
https://github.com/silverstripe/silverstripe-framework/pull/11667#issuecomment-2774092483